recipes-qt/qmllive/qmllive_git.bb

```diff
@@ -0,0 +1,21 @@
+DESCRIPTION = "QmlLive is a live reloader environment for rapid UI development"
+HOMEPAGE = "https://github.com/qtproject/qt-apps-qmllive"
+
+LICENSE = "GPL-3.0"
+LIC_FILES_CHKSUM = " \
+    file://LICENSE.GPL3;md5=75cd0dbc6f2d24e7eeb128ff59b74f4c \
+"
+
+DEPENDS = "qtbase qtdeclarative"
+
+PV = "git${SRCPV}"
+
+SRC_URI = "git://code.qt.io/qt-apps/qmllive.git;branch=dev"
+SRCREV = "0c7bf141b08aa9e757e91a4a05769257d043eab2"
+S = "${WORKDIR}/git"
+
+inherit pkgconfig qmake5
+
+EXTRA_QMAKEVARS_POST = "-after QMAKE_RPATHDIR="
+
+FILES_${PN}-dev += "/usr/share/*"
```

Comment:
Doesn't it respect datadir? And the * isn't necessary.

Reply:
fixed.